vta/tests/hardware/metal_test/device.py

```diff
@@ -0,0 +1,278 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from ctypes import *
+from _macros_h import *
+from insn_lib import *
+import numpy as np
+
+# dictionary that maps actions to their corresponding value
+ACTIONS = {"1DLOAD":VTA_OPCODE_LOAD, "2DLOAD":VTA_OPCODE_LOAD, "ALU":None, "GEMM":None, 
+ "1DSTORE":VTA_OPCODE_STORE, "2DSTORE":VTA_OPCODE_STORE, "FINISH":None}
+# dictionary that maps items to their corresponding ID
+ITEMS = {"UOP":VTA_MEM_ID_UOP, "INP":VTA_MEM_ID_INP, "WGT":VTA_MEM_ID_WGT, "ACC":VTA_MEM_ID_ACC, "OUT":VTA_MEM_ID_OUT,
```

Comment:
watch out for lines that are too long. Dictionary should have 1 line per element